app/assets/stylesheets/blacklight/_facets.scss

```diff
@@ -164,19 +164,18 @@ $facet-toggle-show-icon: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2
 $facet-toggle-hide-icon: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-dash-square" viewBox="0 0 16 16"><path d="M14 1a1 1 0 0 1 1 1v12a1 1 0 0 1-1 1H2a1 1 0 0 1-1-1V2a1 1 0 0 1 1-1h12zM2 0a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V2a2 2 0 0 0-2-2H2z" /><path d="M4 8a.5.5 0 0 1 .5-.5h7a.5.5 0 0 1 0 1h-7A.5.5 0 0 1 4 8z" /></svg>') !default;
 $facet-toggle-width: 1em !default;
 $facet-toggle-height: $facet-toggle-width !default;
+$facet-toggle-margin: 0.5em !default;
```

Comment:
Is it possible to not use a sass variable here?  I'd like to move us away from sass if possible.  Would a css variable work?